lib/install/install.go

```diff
@@ -123,6 +123,21 @@ func (i *Installer) Complete(key ops.SiteOperationKey) error {
 	return trace.Wrap(machine.Complete(trace.Errorf("completed manually")))
 }
 
+// GenerateDebugReport captures the state of the operation to the file given with path.
+// Implements server.DebugReporter
+func (i *Installer) GenerateDebugReport(path string) error {
+	i.WithField("path", path).Info("Generate debug report.")
+	op, err := ops.GetWizardOperation(i.config.Operator)
+	if err != nil {
+		return trace.Wrap(err)
+	}
+	err = i.generateDebugReport(op.ClusterKey(), path)
+	if err != nil {
+		i.WithError(err).Warn("Failed to generate debug report.")
```

Comment:
It's a little weird that this function both logs the error and returns it below. Should the error be logged by the called instead?